src/components/formControls/radio/style.ts

```diff
@@ -0,0 +1,70 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License. v. 2.0. If a copy of the MPL was not distributed with this file.
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled, { css, ThemedStyledProps } from '../../../theme'
+import { Props, Size } from './index'
+
+interface StyleProps {
+  size?: Size
+  disabled?: boolean
+  selected?: boolean
+}
+
+const getThemeColors = (p: ThemedStyledProps<Props>, selected: boolean | undefined) => {
+  let fillColor = '#696FDC'
```

Comment:
Would be great for all colors to come from theme. Can you work with a designer to make sure the color is:
1. From the standard palette
2. Has a semantic name that's appropriate across-controls if applicable, or radio-specific if not.

This is so we can do dark mode if neccessary.

At the very least, if this color would never ever change across themes, then this can come as a variable from palette.js

Reply:
Done, met with @karenkliu and got the correct fill color from the palette.